.github/workflows/recordScreenshots.yml

```diff
@@ -2,18 +2,33 @@ name: Record screenshots
 
 on:
   workflow_dispatch:
+  pull_request:
+    types: [ labeled ]
 
 # Enrich gradle.properties for CI/CD
 env:
   GRADLE_OPTS: -Dorg.gradle.jvmargs="-Xmx3584m -Dfile.encoding=UTF-8 -XX:+HeapDumpOnOutOfMemoryError" -Dkotlin.incremental=false -XX:+UseParallelGC
 
 jobs:
   record:
-    name: Record screenshots on branch ${{ github.ref_name }}
+    name: Record screenshots on branch ${{ github.event.pull_request.head.ref || github.ref_name }}
     runs-on: ubuntu-latest
+    if: github.event_name == 'workflow_dispatch' || github.event.label.name == 'Record-Screenshots'
 
     steps:
-      - name: ⏬ Checkout with LFS
+      - name: Remove Record-Screenshots label
+        if: github.event.label.name == 'Record-Screenshots'
```

Comment:
Does this mean it's going to remove the label as soon as it begins work?
What if we display status with labels too?
It could remove this and add "Screenshot-pending" label while the GH action is ongoing (or maybe it could simply add a comment to the PR with a link to the job?)
And it could add the "Screenshots_recorded" label when finished successfully (and a "Screenshot_error" if something wrong happened?)

Reply:
You can already see all this in the action list at the bottom of the PR, in the list of actions running/finished: while the screenshots are being recorded, a new "Recording screenshots" job is added, and if it fails it will appear there as failed too AFAICT. If it succeeds, you'll just get a new commit in the PR.